### ISSUE TITLE
Change the order of repositories in repository list view

### ISSUE THREAD

**Issue by codian** [desktop/desktop#11608]
I want to change the order of repositories in repository list view.
It would be cool if I could place some repeatedly visited repositories at the top of the list.
It would be also cool if it can be done by drag and drop.

**Comment by yqs112358** [comment]
I was going to post a new Issue, but then I found these similar requests
If the repository sorting and sorting is not done for now, I hope to at least add the repository topping feature
Maybe it is not that hard (?) , like adding a **Pinned** tag before **Recent** and add a context item to pin a repository to **Pinned**

**Comment by Andrej730** [comment]
Currently it's kind of okay if you need to switch between 1-4 different repos (give 3 repos appear at the top in "Recent" section) but pinning/reordeing option would be nice because currently all repositories are just ordered in alphabetical order.

**Comment by n-alonso** [comment]
Yeah! If I want to show my github profile to potential employers, I want my most recent projects on top, not at the bottom of the list.

**Comment by xavierfoucrier** [comment]
Ability to **pin repositories** would be a very useful feature, especially when working with a lot.

**Comment by guplem** [comment]
In case you are willing to switch to a fork, this has been partially addressed in [GitHub Desktop Plus](https://github.com/pol-rivero/github-desktop-plus). It allows you to customize the group a repository belongs to by right-clicking and selecting "Change group name", giving you more control over how repositories are organized in the sidebar.

**Comment by adesai1000** [comment]
You can do this in GlitchHub Desktop, a fork I maintain. Right-click a repository in the list and choose "Move Up" or "Move Down". The order is remembered per group (each GitHub owner, Enterprise host, and "Other"), and repositories you have not moved stay alphabetical below the ones you have. The Recent group is left alone since the app generates that one.

Not affiliated with GitHub. macOS on Apple silicon only right now, and the build is unsigned, so right-click and Open the first time you launch it. It uses its own profile folder.

Download: https://github.com/adesai1000/GlitchHub-Desktop/releases/latest

If it is useful to you, a star on the repo would help a lot.
